hack/log/redact.sh

```diff
@@ -26,7 +26,6 @@ redact_vars=(
     "${AZURE_CLIENT_SECRET:-}"
     "${AZURE_SUBSCRIPTION_ID:-}"
     "${AZURE_TENANT_ID:-}"
-    "${AZURE_JSON_B64:-}"
```

Comment:
We still want to attempt to redact in case one of the templates still uses the old method (but it shouldn't fail if there's nothing to redact.